avm/ptn/ai-platform/baseline/README.md

```diff
@@ -1608,6 +1608,249 @@ The outbound rule. The name of the rule is the object key.
 
 - Required: Yes
 - Type: object
+- Type-deciding property: type
```

Comment:
Waiting for readmes to be regenerated after agreeing on https://github.com/Azure/bicep-registry-modules/pull/4146/files?file-filters%5B%5D=.ps1&file-filters%5B%5D=.psm1&show-viewed-files=true#r1995727675